File: 344-ReverseString/344-ReverseString.js
// Last updated: 3/6/2026, 2:09:19 PM
/**
 * @param {character[]} s
 * @return {void} Do not return anything, modify s in-place instead.
 */
var reverseString = function(s) {
    let i=0;
    let j=s.length-1;

    while(i<j){
        let temp = s[i];
        s[i] = s[j];
        s[j] = temp;
        i++;
        j--;
    }
};

File: 344-ReverseString/344-ReverseString.cpp
// Last updated: 4/9/2026, 10:17:45 AM
1class Solution {
2public:
3    void reverseString(vector<char>& s) {
4        int i=0;
5        int j=s.size()-1;
6
7        while(i<j){
8            int temp = s[i];
9            s[i]=s[j];
10            s[j]=temp;
11            i++;
12            j--;
13        }
14    }
15};